.github/workflows/presubmit.yaml

```diff
@@ -29,11 +28,11 @@ jobs:
       - uses: actions/checkout@v3
         with:
           # This is introduced to make sure 'go generate' works properly
-          path: 'src/k8s-config-connector'
+          path: "src/k8s-config-connector"
```

Comment:
Curious why we want to change to double quote?

Reply:
Ah, VSCode just does this automatically.  We'll be fighting the tools with single quotes.  I can split it out if desired.

Reply:
Gotcha. I use VSCode, but I'm cool with whatever works!